dj_gui_api_server/DJGUIAPIServer.py

```diff
@@ -190,6 +190,7 @@ def get_table_attributes(jwt_payload: dict):
                                                 request.json["schemaName"],
                                                 request.json["tableName"])
     except Exception as e:
+        print(e, flush=True)
```

Comment:
Let's make sure that we only use `app.logger` when logging behavior on the API server side rather than `print`. This to have better control and 'configurability' of the verbosity of server-side logging.

Reply:
stale debug statement, sorry